### PR TITLE
Update messages storage

### PR DESCRIPTION
**Pull request by victorjzsun**

### CODE REVIEW

File: src/js/models/messages.js [hunk 1]
```diff
@@ -1,16 +1,47 @@
...
+			this.contacts = snapshot.docs.map(doc => {
+				const data = doc.data();
+				const pos = this.getPos(data.chatterIds);
+				const otherPos = (pos + 1) % 2;
+				return {
+					id: doc.id,
+					chatterName: data.chatters[otherPos],
+					chatterUid: data.chatterIds[otherPos]
+				}
+			})
```
Comment (Yuxuan-Lin) [inline src/js/models/messages.js:27]: Victor, when rendering contact of a specific conversation, we would need the time and content of the last message in the conversation. Could we also get those two (time and content of the last message) and put it in this.contacts?

Comment (victorjzsun) [reply src/js/models/messages.js:27]: Yes, we'd need to add those two fields to the database as well, but it shouldn't be too big of a change

Comment (victorjzsun) [reply src/js/models/messages.js:27]: Created issue at https://github.com/Yuxuan-Lin/VioLegacy/issues/34

**Review by rg3515 — APPROVED** [review]
I have tested it personally. It works beautifully.

**Pull request merged**